docs/streams/developer-guide/config-streams.html

```diff
@@ -300,7 +305,7 @@ <h4><a class="toc-backref" href="#id23">num.standby.replicas</a><a class="header
               set by the user or all serdes must be passed in explicitly (see also default.key.serde).</td>
             <td><code class="docutils literal"><span class="pre">null</span></code></td>
           </tr>
-          <tr class="row-even"><td>default.windowed.key.serde.inner</td>
+          <tr class="row-even"><td>default.windowed.key.serde.inner (Deprecated.)</td>
```

Comment:
Below is `default.window.value.serde.inner` which was also deprecated, right? (L308 original, new L313)